### DISCUSSION TITLE
脚本征集

### DISCUSSION THREAD

**Discussion by OpportunityLiu** [EhTagTranslation/UserScripts#2] [Ideas]
征集可以放在这里的用户脚本

**Comment by OpportunityLiu** [comment]
@xioxin 之前ETS的部分功能可以拆分出来在这个repo底下建文件夹存储，比如磁力链的那个。

**Comment by Mapaler** [comment]
我把ETS移植到一半，你把文件给删了🙄

**Comment by Mapaler** [comment]
那我姬需在原来的仓库上修改ETS吧，只是让其改为使用迁移后的数据库。

**Comment by OpportunityLiu** [comment]
ETS还改啥啊

**Comment by Mapaler** [comment]
> 
> 
> ETS还改啥啊

只是让其改为使用迁移后的数据库，就是最后的兼容性支持。还想继续用ETS的，至少能用上新的数据库。

**Comment by Mapaler** [comment]
@xioxin 你要不把你那个 [阅读状态](https://github.com/EhTagTranslation/UserScripts/tree/master/ReadStatus) 改成 CSS 吧（毕竟我没你的 Greasy Fork 权限）， Greasy Fork 上可以自由选择安装成脚本还是安装成样式。 参考这个 <https://sleazyfork.org/scripts/406417> ，代码我都已经给你转好css文件了的。

![图像 41](https://user-images.githubusercontent.com/6565860/103140803-30eb7a00-4726-11eb-8962-5f87cc91752f.png)
![图像 38](https://user-images.githubusercontent.com/6565860/103140806-3a74e200-4726-11eb-87de-b32351eddd32.png)
![图像 40](https://user-images.githubusercontent.com/6565860/103140807-3c3ea580-4726-11eb-8fcf-4431c3e56ae5.png)

**Comment by xioxin** [comment]
![image](https://user-images.githubusercontent.com/5716100/103143854-f4397600-4759-11eb-82c0-fa38c1a7ad19.png)

**Comment by Mapaler** [comment]
@xioxin 是不是需要新建一个样式才行
![图像 42](https://user-images.githubusercontent.com/6565860/103147008-9a51a400-478b-11eb-82f8-aac748954d13.png)
